cmd/internal/cli/pull.go

```diff
@@ -48,6 +49,8 @@ var (
 	KeyServerURL = "https://keys.sylabs.io"
 	// unauthenticatedPull when true; wont ask to keep a unsigned container after pulling it
 	unauthenticatedPull bool
+	// PullPath is the path that the containers will be pulled to, if set
+	PullPath string
```

Comment:
Thoughts on changing this to `PullDir`? I agree with this comment: https://github.com/sylabs/singularity/issues/2788#issuecomment-471322723 saying that path implies a full file path while this is looking for a predefined directory to place the image

Reply:
Sure! I'll fix it :+1: